torchmultimodal/utils/preprocess.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+#
+# preprocess.py
```

Comment:
Since the logic here is made modality agnostic and the only consumer is the quantization layer (for now), I'm wondering if we need a separate module for them. Alternatively, we can also consider `torchmultimodal/transforms/`, or the kitchen sink `torchmultimodal/utils/common.py`.

@ankitade Any thoughts here?

Reply:
@langong347 thoughts on point 3?

Reply:
Agreed on (3) we can move them under `Quantisation` class.

Reply:
also wanted to add that if the flattening and unflattening is done in the quantisation layer it's more straightforward because I don't have to feed the `permuted_shape` to unflatten it anymore

Reply:
3 sounds good. forward will call these utils iiuc